cli/fathom_web/commands/test.py

```diff
@@ -0,0 +1,77 @@
+from json import JSONDecodeError, load, loads
+
+from click import argument, BadParameter, command, File, style
+
+from ..accuracy import accuracy_per_tag, accuracy_per_page, pretty_accuracy
+from ..utils import classifier, tensor, tensors_from
+
+
+def decode_weights(ctx, param, value):
+    """Validate a click option, making sure it's a valid JSON object with
+    properly formatted "coeff" and "bias" keys."""
+    try:
+        decoded_weights = loads(value)
+    except JSONDecodeError:
+        raise BadParameter('Weights must be a valid JSON object.')
+    else:
+        if 'coeffs' in decoded_weights and 'bias' in decoded_weights:
+            if type(decoded_weights['bias']) is float:
```

Comment:
I believe the best practice for type checking is through `isinstance()` ([this](https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance) is my goto post on the topic) instead of `type` unless you want to be checking for exact types.

Reply:
You are correct! Sorry, my JS is seeping back into my Python. Eek!